src/anders_karlsen_ex/pa02/snakes_simulation.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Anders Karlsen", "Kåre Johnsen"
+__email__ = "anderska@nmbu.no", "kajohnse@nmbu.no"
+
+import random
+from collections import Counter
+
+
+class Board:
+    """
+    Defining the board with its snakes, ladders and the goal.
+    """
+    def __init__(self, *args, **kwargs):
+        self.extra = args, kwargs
+        self.ladders = {1: 40, 8: 10, 36: 52, 43: 62, 49: 79, 65: 82, 68: 85}
+        self.snakes = {25: 5, 33: 3, 42: 30, 56: 37, 64: 27, 74: 12, 87: 70}
+        self.resilient_move = False
```

Comment:
Those variables are related to the player not the board, and also more than one player share the same board